content/ko/docs/setup/production-environment/windows/user-guide-windows-nodes.md

```diff
@@ -0,0 +1,271 @@
+| 쿠버네티스 DNS 서비스 IP | DNS 확인 및 클러스터 서비스 검색에 사용되는 서비스인 `kube-dns`의 IP 주소이다. | 10.96.0.10 |
+
+클러스터에 IP 주소를 얼마나 할당해야 할지 결정하기 위해 '쿠버네티스에서 윈도우 컨테이너: 지원되는 기능: 네트워킹'에서 소개한 네트워킹 선택 사항을 검토하자.
+
+### 윈도우를 실행되는 구성 요소
+
+쿠버네티스 컨트롤 플레인이 리눅스 노드에서 운영되는 반면, 다음 요소는 윈도우 노드에서 구성되고 운영된다.
+
+1. kubelet
+2. kube-proxy
+3. kubectl (선택적)
+4. 컨테이너 런타임
+
+v1.14 이후의 최신 바이너리를 [https://github.com/kubernetes/kubernetes/releases](https://github.com/kubernetes/kubernetes/releases)에서 받아온다.  kubeadm, kubectl, kubelet, kube-proxy의 Windows-amd64 바이너리는 CHANGELOG 링크에서 찾아볼 수 있다.
+
+### 네트워크 구성
+
+리눅스 기반의 쿠버네티스 마스터 노드를 가지고 있다면 네트워킹 솔루션을 선택할 준비가 된 것이다. 이 가이드는 단순화를 위해 VXLAN 방식의 플라넬(Flannel)을 사용하여 설명한다.
+
+#### 리눅스 컨트롤러에서 VXLAN 방식으로 플라넬 구성하기
+
+1. 플라넬을 위해 쿠버네티스 마스터를 준비한다.
+
+    클러스터의 쿠버네티스 마스터에서 사소한 준비를 권장한다. 플라넬을 사용할 때에 iptables 체인으로 IPv4 트래픽을 브릿지할 수 있게 하는 것은 추천한다. 이는 다음 명령어를 이용하여 수행할 수 있다.
+
+    ```bash
+    sudo sysctl net.bridge.bridge-nf-call-iptables=1
+    ```
+
+1. 플라넬 다운로드 받고 구성하기
+
+    가장 최신의 플라넬 메니페스트를 다운로드한다.
+
+    ```bash
+    wget https://raw.githubusercontent.com/coreos/flannel/master/Documentation/kube-flannel.yml
+    ```
+
+    VXLAN 네트워킹 벡엔드를 가능하게 하기 위해 수정할 곳은 두 곳이다.
+
+    아래 단계를 적용하면 `kube-flannel.yml`의 `net-conf.json`부분을 다음과 같게 된다.
+
+    ```json
+    net-conf.json: |
+        {
+          "Network": "10.244.0.0/16",
+          "Backend": {
+            "Type": "vxlan",
+            "VNI" : 4096,
+            "Port": 4789
+          }
+        }
+    ```
+
+    {{< note >}}리눅스의 플라넬과 윈도우의 플라넬이 상호운용하기 위해서 `VNI`는 반드시 4096이고, `Port`는 4789여야 한다. 다른 VNI는 곧 지원한다. [VXLAN 문서](https://github.com/coreos/flannel/blob/master/Documentation/backends.md#vxlan)에서
+     이 필드의 설명 부분을 보자.{{< /note >}}
+
+1. `kube-flannel.yml`의 `net-conf.json` 부분을 거듭 확인하자.
+    1. 클러스터 서브넷(예, "10.244.0.0/16")는 IP 주소 설계에 따라 설정되어야 한다.
+        * VNI 4096 은 벡엔드에 설정한다.
+        * Port 4789 은 벡엔드에 설정한다.
+    2. `kube-flannel.yml`의 `cni-conf.json` 부분에서 네트워크 이름을 `vxlan0`로 바꾼다.
+
+
+    `cni-conf.json`는 다음과 같다.
+
+    ```json
+    cni-conf.json: |
+        {
+          "name": "vxlan0",
+          "plugins": [
+            {
+              "type": "flannel",
+              "delegate": {
+                "hairpinMode": true,
+                "isDefaultGateway": true
+              }
+            },
+            {
+              "type": "portmap",
+              "capabilities": {
+                "portMappings": true
+              }
+            }
+          ]
+        }
+    ```
+
+1. 플라넬 yaml 을 적용하고 확인하기
+
+    플라넬 구성을 적용하자.
+
+    ```bash
+    kubectl apply -f kube-flannel.yml
+    ```
+
+    다음은 플라넬 파드는 리눅스 기반이라 [여기](https://github.com/Microsoft/SDN/blob/1d5c055bb195fecba07ad094d2d7c18c188f9d2d/Kubernetes/flannel/l2bridge/manifests/node-selector-patch.yml)나온 `NodeSelector` 패치를 플라넬 데몬셋 파드에 적용한다.
```

Comment:
[원문]
Next, since the Flannel pods are Linux-based, apply a NodeSelector patch, which can be found [here](https://github.com/Microsoft/SDN/blob/1d5c055bb195fecba07ad094d2d7c18c188f9d2d/Kubernetes/flannel/l2bridge/manifests/node-selector-patch.yml), to the Flannel DaemonSet pod:

일단, 원문을 고려하여 다음과 같이 의견 드립니다.
```suggestion
    다음은 플라넬 파드는 리눅스 기반이라 [여기](https://github.com/Microsoft/SDN/blob/1d5c055bb195fecba07ad094d2d7c18c188f9d2d/Kubernetes/flannel/l2bridge/manifests/node-selector-patch.yml) 나온 노드 셀렉터 패치를 플라넬 데몬셋 파드에 적용한다.
```